lib/glific_web/providers/maytapi/controller/status_controller.ex

```diff
@@ -0,0 +1,22 @@
+defmodule GlificWeb.Providers.Maytapi.Controllers.StatusController do
+  @moduledoc false
+
+  use GlificWeb, :controller
+  alias Glific.WAManagedPhones
+
+  @doc false
+  @spec handler(Plug.Conn.t(), map(), String.t()) :: Plug.Conn.t()
+  def handler(conn, _params, _msg) do
+    conn
+    |> Plug.Conn.send_resp(200, "")
+    |> Plug.Conn.halt()
+  end
+
+  @doc false
+  @spec status(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def status(conn, %{"status" => status, "phone_id" => phone_id} = params) do
+    WAManagedPhones.status(status, phone_id)
```

Comment:
Can we put this function inside a `Task`, you can check how we do it in `lib/glific_web/providers/maytapi/controller/message_event_controller.ex:22`. Why we do this because, thirdy party apis expect servers to return a 200 asap, so that they can decide they should retry or not.